### ISSUE TITLE
Syncthing for CentOS and ClearOS

### ISSUE THREAD

**Issue by marclaporte** [onuralpszr/syncthing_rpm#3]
Hi!

Would it be possible to have a solution for CentOS and ClearOS?

Ideally for v6 and v7

Thanks!

**Comment by onuralpszr** [comment]
Well I'm not using CentOS or ClearOS but maybe CentOS I can work with it.

**Comment by onuralpszr** [comment]
I created for CentOS 7 x64 . Reason is  CentOS 7 didn't have x86 and I will work on CentOS 6 for both ARCH and that should be fine. And please test the package send feedback and If I you don't have CentOS 7 at the moment please tell me I will create for CentOS 6 too.

Thank you

https://dl.dropboxusercontent.com/u/17305215/syncthing-0.9.15-1.9.el7.centos.x86_64.rpm

**Comment by marclaporte** [comment]
About ClearOS: if we have CentOS, it will almost certainly work fine with ClearOS. I will document the bit about opening up the firewall port (by default, one of the ClearOS modes has a firewall that blocks almost everything). So don't worry about this.

About CentOS. I tested v7. The RPM installed fine. And on the first run, it created the keys & all. However, on first or subsequent runs, it doesn't start the browser.

When I visit the URL manually, I get the image below:
![screenshot_centos7_2014-09-13-syncthing](https://cloud.githubusercontent.com/assets/1004261/4261313/38d48b68-3b6a-11e4-9eb6-009f280afe4c.png)

Thanks!

**Comment by marclaporte** [comment]
And clicking "Add Node" or "Add Repository" doesn't work.

Thanks!

**Comment by onuralpszr** [comment]
Could you send logs and I'll look into it. 

Thank you.

**Comment by onuralpszr** [comment]
Well main problem was "Konqueror" browser. If you just install "firefox" It will be fine as you can see it. Simply lack of support on "Konqueror" It even asked for github "js" support and "old" browser title so It simply asking for better browser and that's all.

![snapshot1](https://cloud.githubusercontent.com/assets/1688848/4265783/acab00b2-3c5f-11e4-9a93-6e5a35e3181d.png)

**Comment by marclaporte** [comment]
Ok, thanks, so the same RPM for v6?

**Comment by onuralpszr** [comment]
No. It's not Because centos 6 was using sysvinit so I need to change systemd part then create new package we gonna need it

**Comment by marclaporte** [comment]
Hi again!

Any news on CentOS 6.x?

FYI, I installed another Fedora 20 and it worked well. It installed an older version, but it was a 1-click update.

Thanks!

**Comment by marclaporte** [comment]
Related links:
https://pulse-forum.ind.ie/t/centos6-x86-64-rpm/981
https://pulse-forum.ind.ie/t/keeping-syncthing-running-systemd-regular-etc-init-d/402

**Comment by marclaporte** [comment]
Related link:
https://discourse.syncthing.net/t/getting-syncthing-running-with-a-central-file-server-running-centos-7-minimal-setup-and-windows-clients/1429

Thanks!

**Comment by marclaporte** [comment]
Related link about "centos6 x86_64 rpm"
https://discourse.syncthing.net/t/centos6-x86-64-rpm/981

Thanks!
